daemon/algod/api/client/restClient.go

```diff
@@ -559,7 +560,9 @@ func (client RestClient) SendRawTransactionGroup(txgroup []transactions.SignedTx
 }
 
 // Block gets the block info for the given round
-func (client RestClient) Block(round uint64) (response model.BlockResponse, err error) {
+func (client RestClient) Block(round uint64) (response v2.BlockResponseJSON, err error) {
```

Comment:
For `Block` and the other state delta endpoints, I opted to decode the responses into the struct types that are used for encoding in `daemon/algod/api/server/v2/handlers.go`. Otherwise these objects are thin wrappers around `map[string]interface{}`, which is not as useful.